go/signedexchange/cmd/gen-signedexchange/main.go

```diff
@@ -42,7 +42,8 @@ var (
 	flagDate           = flag.String("date", "", "The datetime for the signed exchange in RFC3339 format (2006-01-02T15:04:05Z07:00). Use now by default.")
 	flagExpire         = flag.Duration("expire", 1*time.Hour, "The expire time of the signed exchange")
 
-	flagDumpSignatureMessage = flag.String("dumpSignatureMessage", "", "Dump signature message bytes to file for debugging.")
+	flagDumpSignatureMessage = flag.String("dumpSignatureMessage", "", "Dump signature message bytes to a file for debugging.")
+	flagDumpHeadersCbor      = flag.String("dumpHeadersCbor", "", "Dump metadata and headers encoded as a canonical CBOR to a file for debugging.")
```

Comment:
I realized that the flag names include upper and lower cases... I was wondering if this is usual 🤔